Demo/Demo/FeatureViewControllers/PaymentButtonCustomizationViewController.swift

```diff
@@ -125,6 +133,12 @@ class PaymentButtonCustomizationViewController: UIViewController {
         if sender == fundingPicker {
             reloadColorPicker()
         }
+        let fundingSource = PaymentButtonFundingSource.allCases()[fundingPicker.selectedSegmentIndex]
+        if fundingSource == .payPal {
+            labelPicker.isHidden = false
+        } else {
+            labelPicker.isHidden = true
+        }
```

Comment:
```suggestion
labelPicker.isHidden = fundingSource != .payPal
```

Reply:
+1 - this enum should be able to be updated to:
```
    public enum Label: String, CaseIterable {
        /// Display no label
        case none
        
        /// Display "Checkout" on the right side of the button's logo
        case checkout = "Checkout"

        /// Display "Buy now" on the right side of the button's logo
        case buyNow = "Buy Now"

        /// Display "Pay with" on the left side of the button's logo
        case payWith = "Pay with"

        var label: PaymentButtonLabel? {
            PaymentButtonLabel(rawValue: rawValue)
        }
        
        public static func allCasesAsString() -> [String] {
            Self.allCases.map { $0.rawValue }
        }
    }
```

Reply:
@jaxdesmarais i just realized this shouldnt be here. We had the same for the other enums of the buttons but we removed. This should be in an extension on the Demo app